<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExistingRecipe</class>
 <widget class="QDialog" name="ExistingRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Existing Recipe</string>
  </property>
  <widget class="QLabel" name="CurrentDateLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ExistingRecipeStatusLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>200</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>330</height>
    </size>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ContiguousSelection</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="FatLabel">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Total Fat:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="ProtienInputLabel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="CaloriesInputLabel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="CalorieLabel">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Total Calories:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ProtienLabel">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Total Protien:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="CarbsLabel">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Total Carbs:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="FatInputLabel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="CarbsInputLabel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="CloseWindowButton">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>275</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddRecipeCurrentButton">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>275</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
